code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -186,13 +186,15 @@
 	..()
 
 /mob/living/carbon/human/proc/Stasis(amount)
-	if((species.flags & NO_SCAN) || isSynthetic())
+	//if((species.flags & NO_SCAN) || isSynthetic()) vorestation removal
+	if(((species.flags & NO_SCAN) && !istype(species, /datum/species/xenochimera)) || isSynthetic()) //vorestation addition

```

Comment:
Oh shit, I didn't know about that one. That probably makes a lot more sense to use.
Looking over it, the only other places it's set are monkeying, gibbing and staff of change, and purely to stop the mob moving while they do their thing.